src/atoms/author/author.tsx

```diff
@@ -35,8 +36,15 @@ export const getAuthorNames = (authors: AuthorTypesStory[]) =>
  * @param {AuthorProps} props
  * @param {Author[]} props.authors Array containing details about all authors of the story. Comes from Story API
  * @param {(string | React.Component)} props.prepend Optional. Used to prepend either some string or a component containing some icon to authors.
+ *
  */
-const Author = ({ authors, prepend }: AuthorProps) => {
+const Author = ({ authors, prepend, story, config, storyType, theme }: AuthorProps) => {
+  const getAuthorCard = get(config, ["opts", "render", "authorCardRender"], null);
+
+  if (getAuthorCard && typeof getAuthorCard === "function") {
+    return getAuthorCard({ story, config, storyType, theme });
```

Comment:
no need of passing `storyType`, we can get it from the `story` object itself

Reply:
done